java-components/build-recipes-database/src/test/resources/test-recipes/build-info/github.com/lz4/lz4-java/build.yaml

```diff
@@ -0,0 +1,18 @@
+---
+enforceVersion: true
+additionalArgs:
+  - "-Dfoo=foo"
+preBuildScript: |
+    sed -i -e 's|2\.7\.8|2.5.3|g;' -e 's|-SNAPSHOT|.0|g;' ivy.xml ivy-pure-java.xml
+postBuildScript: |
+    mvn -B -e -s "$(workspaces.build-settings.path)/settings.xml" org.apache.maven.plugins:maven-deploy-plugin:3.1.0:deploy-file "-Durl=file:$(workspaces.source.path)/artifacts" -DpomFile=dist/lz4-java-$(params.ENFORCE_VERSION).pom -Dfile=dist/lz4-java-$(params.ENFORCE_VERSION).jar -Dfiles=dist/lz4-java-$(params.ENFORCE_VERSION)-sources.jar,dist/lz4-java-$(params.ENFORCE_VERSION)-javadoc.jar -Dclassifiers=sources,javadoc -Dtypes=jar,jar
+additionalDownloads:
```

Comment:
This illustrates the problem with merging. The pure java version does not need this, but there is no way to un-require it if they are merged.

Reply:
The pure-java version just generates different files to the dist directory - without this deploy command nothing gets deployed I thought?

Reply:
I mean the additionalDownloads section, the java version should not need the native dependencies.